specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -6956,6 +6956,44 @@
       "required": [
         "endpoint"
       ]
+    },
+    "PowerBILinkedService": {
+      "x-ms-discriminator-value": "PowerBI",
+      "description": "Power BI linked service.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/LinkedService"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Power BI linked service properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PowerBILinkedServiceTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "PowerBILinkedServiceTypeProperties": {
+      "description": "Power BI linked service properties.",
+      "type": "object",
+      "properties": {
+        "workspaceId": {
+          "type": "string",
+          "description": "The Power BI workspace id."
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "The tenant id to which the Power BI workspace belongs."
+        }
+      },
+      "required": [
+        "workspaceId",
+        "tenantId"
+      ]
```

Comment:
ADF doesn't support PBI linked service, it's synapse only.

Reply:
removed.